code/control/DNRoot.php

```diff
@@ -1370,10 +1370,11 @@ public function deploySummary(SS_HTTPRequest $request) {
 		);
 		$data = $strategy->toArray();
 
-		// Add in a URL for comparing from->to code changes.
+		// Add in a URL for comparing from->to code changes. Ensure that we have
+		// two proper 40 character SHAs, otherwise we can't show the compare link.
```

Comment:
Is 1.2..1.3 not a valid comparison? Will from & to always be 40-char sha's? 

Why does this fail anyway, what's the trigger?